src/app/backend/handler/apihandler.go

```diff
@@ -735,15 +570,21 @@ func (apiHandler *APIHandler) handleGetRbacRoleBindingList(request *restful.Requ
 
 func (apiHandler *APIHandler) handleGetCsrfToken(request *restful.Request, response *restful.Response) {
 	action := request.PathParameter("action")
-	token := xsrftoken.Generate(apiHandler.csrfKey, "none", action)
-	response.WriteHeaderAndEntity(http.StatusOK, CsrfToken{Token: token})
+	token := xsrftoken.Generate(apiHandler.manager.CSRFKey(), "none", action)
+	response.WriteHeaderAndEntity(http.StatusOK, api.CsrfToken{Token: token})
 }
 
 func (apiHandler *APIHandler) handleGetStatefulSetList(request *restful.Request, response *restful.Response) {
+	k8sClient, err := apiHandler.manager.Client(request)
+	if err != nil {
```

Comment:
I am wondering, what we can do with this error check to avoid it within every handler. There is an option of doing all error handling while getting client, but it seems like a wrong place. I guess, we will leave it here, since handler is the best place to work with requests and responses. If you can see any options how to get rid of it it, go for it.